### PR TITLE
Adds BooleanColumn to builtin columns

### PR DESCRIPTION
**Pull request by lisad**

### CODE REVIEW

File: phaser/column.py [hunk 1]
```diff
@@ -149,6 +149,41 @@ def fix_value(self, value):
...
 
 
+class BooleanColumn(Column):
+    TRUE_VALUES = ['t', 'true', '1', 'yes', 'y']
+    FALSE_VALUES = ['f', 'false', '0', 'no', 'n']
+
+    def __init__(self,
+                 name,
+                 required=True,
+                 null=False,  #Test this being either way
```
Comment (jeffkole) [inline phaser/column.py:159]: I am starting to think that the parameter should be named something like `null_allowed`.  I can't keep straight in my head what the `null` parameter means.

File: tests/test_columns.py [hunk 2]
```diff
@@ -145,6 +145,34 @@ def test_curious_quote_situation(tmpdir):
...
+def test_boolean_required():
+    phase = Phase(columns=[BooleanColumn("test", required=True)])
+    phase.load_data([{'id': 1}])
+    with pytest.raises(DataErrorException):
+        phase.do_column_stuff()
+
+
+def test_boolean_not_null():
+    with pytest.raises(DataErrorException):
+        BooleanColumn('test', null=False).check_and_cast_value({'id': 1, 'present': None})
```
Comment (jeffkole) [inline tests/test_columns.py:174]: Do you want to also add a test for when `null=True`?

**Review by jeffkole — APPROVED** [review]
